packages/block-library/src/social-link/index.php

```diff
@@ -306,6 +306,10 @@ function block_core_social_link_services( $service = '', $field = '' ) {
 			'name' => 'Share Icon',
 			'icon' => '<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9 11.8l6.1-4.5c.1.4.4.7.9.7h2c.6 0 1-.4 1-1V5c0-.6-.4-1-1-1h-2c-.6 0-1 .4-1 1v.4l-6.4 4.8c-.2-.1-.4-.2-.6-.2H6c-.6 0-1 .4-1 1v2c0 .6.4 1 1 1h2c.2 0 .4-.1.6-.2l6.4 4.8v.4c0 .6.4 1 1 1h2c.6 0 1-.4 1-1v-2c0-.6-.4-1-1-1h-2c-.5 0-.8.3-.9.7L9 12.2v-.4z"/></svg>',
 		),
+		'x'             => array(
```

Comment:
It might be a good idea to add after the `whatsapp` to alphabetize, although `share` doesn't follow that rule.

Reply:
👍 Done in fd299b471c387dedc19f9868377b560a3e006d7c